src/http/ngx_http_mruby_async.c

```diff
@@ -149,7 +149,7 @@ static void ngx_mrb_async_sleep_cleanup(void *data)
 
 static mrb_value ngx_mrb_async_sleep(mrb_state *mrb, mrb_value self)
 {
-  unsigned int timer;
+  mrb_int timer;
```

Comment:
[IMO] negative check required between `mrb_get_args` and casting to `ngx_msec_t` at:

https://github.com/matsumotory/ngx_mruby/pull/351/files#diff-bbdc89dae16aa5c53566f7ec784a35b3R183

Because `ngx_msec_t` looks unsigned, so when `mrb_int timer` got to be negative, it looks dangerous

```c
typedef ngx_rbtree_key_t      ngx_msec_t;
typedef ngx_uint_t  ngx_rbtree_key_t;
```